py-scripts/lf_interop_video_streaming.py

```diff
@@ -317,41 +347,290 @@ def start_specific(self,cx_start_list):
     #     for endpoint in self.generic_endps_profile.created_endp:
     #         self.generic_endps_profile.set_report_timer(endp_name=endpoint, timer=250)
 
+    def map_sta_ips_real(self, sta_list=None):
+        if sta_list is None:
+            sta_list = []
+        for sta_eid in sta_list:
+            eid = self.name_to_eid(sta_eid)
+            sta_list = self.json_get("/port/%s/%s/%s?fields=alias,ip" % (eid[0], eid[1], eid[2]))
+            # print("map_sta_ips - sta_list:{sta_list}".format(sta_list=sta_list))
+            '''
+            sta_list_tmp = self.json_get("/port/%s/%s/%s?fields=ip" % (eid[0], eid[1], eid[2]))
+            print("map_sta_ips - sta_list_tmp:{sta_list_tmp}".format(sta_list_tmp=sta_list_tmp))
+            '''
+            if sta_list['interface'] is not None:
+                # print("map_sta_ips - sta_list_2:{sta_list_2}".format(sta_list_2=sta_list['interface']))
+                # self.ip_map[sta_list['interface']['alias']] = sta_list['interface']['ip']
+                eid_key = "{eid0}.{eid1}.{eid2}".format(eid0=eid[0], eid1=eid[1], eid2=eid[2])
+                self.ip_map[eid_key] = sta_list['interface']['ip']
+
+    def create_real(self, ports=None, sleep_time=.5, debug_=False, suppress_related_commands_=None, http=False, ftp=False, real=False,
+                    https=False, user=None, passwd=None, source=None, ftp_ip=None, upload_name=None, http_ip=None,
+                    https_ip=None, interop=None, media_source=None, media_quality=None, timeout=10, proxy_auth_type=0x2200, windows_list=[], get_url_from_file=False):
+        if ports is None:
+            ports = []
+        cx_post_data = []
+        # print("http_profile - ports:{ports}".format(ports=ports))
+        self.map_sta_ips_real(ports)
+        logger.info("Create HTTP CXs..." + __name__)
+        # print("http_profile - self.ip_map:{ip_map}".format(ip_map=self.ip_map))
+
+        for i in range(len(list(self.ip_map))):
+            url = None
+            if i != len(list(self.ip_map)) - 1:
+                port_name = list(self.ip_map)[i]
+                ip_addr = self.ip_map[list(self.ip_map)[i + 1]]
+            else:
+                port_name = list(self.ip_map)[i]
+                ip_addr = self.ip_map[list(self.ip_map)[0]]
+
+            if (ip_addr is None) or (ip_addr == ""):
+                raise ValueError("HTTPProfile::create encountered blank ip/hostname")
+            if interop:
+                if list(self.ip_map)[i] in windows_list:
+                    self.dest = 'NUL'
+                if list(self.ip_map)[i] not in windows_list:
+                    self.dest = '/dev/null'
+
+            # print("http_profile - port_name:{port_name}".format(port_name=port_name))
+            rv = self.local_realm.name_to_eid(port_name)
+            # print("http_profile - rv:{rv}".format(rv=rv))
+            '''
+            shelf = self.local_realm.name_to_eid(port_name)[0]
```

Comment:
Quite a bit of commented out debugging code. Again good opportunity to use `logger.debug()`. No change required.